packages/3id-did-resolver/src/legacyResolver.ts

```diff
@@ -11,11 +12,17 @@ interface IPFS {
 
 // Legacy 3ids available from 3box, other v1 will always be resolved through ipfs and other services 
 const THREEBOX_API_URL = 'https://ipfs.3box.io'
+const LIMIT = 100
+const fetchCache = new LRUMap<string, any>(LIMIT)
 
 const fetchJson = async (url: string): Promise<any> =>  {
+  const cached = fetchCache.get(url)
+  if (cached) return cached
```

Comment:
can v0 3ids ever be updated?  Do we need to worry about cache invalidation at all?

Reply:
They can not. They are static, so no cache invalidation.

Reply:
Nope! 3IDv0 are just static ipfs objects.